src/main/kotlin/calculator/Formula.kt

```diff
@@ -0,0 +1,33 @@
+package calculator
+
+class Formula(input: String) {
+    private val operands: List<Operand>
+
+    init {
+        val customDelimiterAndOperands = input.split("\n")
+        val customDelimiter: String? = getCustomDelimiter(customDelimiterAndOperands)
```

Comment:
```suggestion
        val customDelimiter = getCustomDelimiter(customDelimiterAndOperands)
```
타입은 생략되어도 괜찮을 것 같습니다!